app/controllers/epp_controller.rb

```diff
@@ -407,4 +400,26 @@ def session_timeout_reached?
     timeout = 5.minutes
     epp_session.updated_at < (Time.zone.now - timeout)
   end
+
+  def iptables_counter_update
+    return if ENV['iptables_counter_enabled'].blank? && ENV['iptables_counter_enabled'] != 'true'
+    return if current_user.blank?
```

Comment:
That's also a bit weird, `current_user` shouldn't be a `String`, it should be an instance of `User` or `Nil`.
`return unless current_user` should communicate the intent better.

Reply:
I don't a see the point in improving code structure. I think the whole feature should be refactored, which requires some effort and is definitely a topic of another branch/ticket.